syft/pkg/cataloger/cataloger.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/anchore/syft/syft/artifact"
 	"github.com/anchore/syft/syft/pkg"
 	"github.com/anchore/syft/syft/pkg/cataloger/apkdb"
+	"github.com/anchore/syft/syft/pkg/cataloger/dartlang"
```

Comment:
Question: is `dartlang` or `dart` more idiomatic? (I do think I prefer your choice of `dartlang` here)

Reply:
I was following the paradigm that the golang module set.

Reply:
`dart` is probably a better pick. One reason going with just `go` for the go modules was that `go` is a reserved word.